packages/arb-token-bridge-ui/src/util/deposits/helpers.ts

```diff
@@ -464,65 +465,71 @@ export async function fetchTeleporterDepositStatusData({
   }
 }
 
-export const getL1ToL2MessageDataFromL1TxHash = async ({
+export const getParentToChildMessageDataFromParentTxHash = async ({
   depositTxId,
   isEthDeposit,
-  l1Provider,
-  l2Provider,
+  parentProvider,
+  childProvider,
   isClassic // optional: if we already know if tx is classic (eg. through subgraph) then no need to re-check in this fn
 }: {
   depositTxId: string
-  l1Provider: Provider
+  parentProvider: Provider
   isEthDeposit: boolean
-  l2Provider: Provider
+  childProvider: Provider
   isClassic?: boolean
 }): Promise<{
   isClassic?: boolean
-  l1ToL2Msg?:
+  parentToChildMsg?:
     | ParentToChildMessageReaderClassic
     | EthDepositMessage
     | ParentToChildMessageReader
 }> => {
   // fetch L1 transaction receipt
-  const depositTxReceipt = await l1Provider.getTransactionReceipt(depositTxId)
+  const depositTxReceipt = await parentProvider.getTransactionReceipt(
+    depositTxId
+  )
 
   // TODO: Handle tx not found
   if (!depositTxReceipt) {
     return {}
   }
 
-  const l1TxReceipt = new ParentTransactionReceipt(depositTxReceipt)
+  const parentTxReceipt = new ParentTransactionReceipt(depositTxReceipt)
 
   const getClassicDepositMessage = async () => {
-    const [l1ToL2Msg] = await l1TxReceipt.getParentToChildMessagesClassic(
-      l2Provider
-    )
+    const [parentToChildMsg] =
+      await parentTxReceipt.getParentToChildMessagesClassic(childProvider)
     return {
       isClassic: true,
-      l1ToL2Msg: l1ToL2Msg
+      parentToChildMsg: parentToChildMsg
```

Comment:
```suggestion
      parentToChildMsg
```

Reply:
we are supposed to have eslint to tell us about this

but we haven't added out shared eslint to the repo